packages/perseus-editor/src/widgets/radio/editor.tsx

```diff
@@ -301,50 +295,42 @@ class RadioEditor extends React.Component<RadioEditorProps> {
         );
         return (
             <div>
+                <Link
+                    href="https://www.khanacademy.org/internal-courses/content-creation-best-practices/xe46daa512cd9c644:question-writing/xe46daa512cd9c644:multiple-choice/a/stems"
+                    target="_blank"
+                >
+                    Multiple choice best practices
+                </Link>
                 <div className="perseus-widget-row">
-                    <a
-                        href={
-                            // This is an editor component, not user-facing.
-                            "https://www.khanacademy.org/internal-courses/content-creation-best-practices/xe46daa512cd9c644:question-writing/xe46daa512cd9c644:multiple-choice/a/stems"
-                        }
-                        target="_blank"
-                        rel="noreferrer"
-                    >
-                        Multiple choice best practices
-                    </a>
-                    <br />
-                    <div className="perseus-widget-left-col">
-                        <Checkbox
-                            label="Multiple selections"
-                            checked={this.props.multipleSelect}
+                    <LabeledSwitch
+                        label="Randomize order"
+                        checked={this.props.randomize}
+                        onChange={(value) => {
+                            this.props.onChange({randomize: value});
+                        }}
+                        style={{marginBottom: spacing.xxSmall_6}}
+                    />
+                    <LabeledSwitch
+                        label="Multiple selections"
+                        checked={this.props.multipleSelect}
+                        onChange={(value) => {
+                            this.onMultipleSelectChange({
+                                multipleSelect: value,
+                            });
+                        }}
+                        style={{marginBottom: spacing.xxSmall_6}}
```

Comment:
Iirc, I think we want to use sizing wherever possible, as spacing is going away eventually.